harness/determined/common/experimental/model.py

```diff
@@ -12,45 +12,54 @@ class ModelVersion:
     """
     A ModelVersion object includes a Checkpoint,
     and can be fetched using ``model.get_version()``.
+
+    Attributes:
```

Comment:
good point, done.

Reply:
Thanks. This goes most of the way there for me, but I think to offer a concrete suggestion (and plagiarizing, partly, from the docs):

```
"""A class representing a combination of Model and Checkpoint.

Once a model has been added to the registry, checkpoints can be added to it. These registered checkpoints are ModelVersions.
```

The attribute section... isn't great (also not your fault).
* What's the difference between a `model_version` and a `model_version_id`? Here's a possible string for `model_version`, also paraphrased from docs: "model_version is an int assigned by the registry. They start at 1 and increment each time a new model version is registered."
* Especially for some of the non-primative attributes, a type might be helpful. What exactly _is_ metadata? What should I as a user of `ModelVersion` know? (type is probably sufficient for me to look it up)

And I guess now that we're talking about it, should `model_id` and `model_verison_id` be private? Not stored in a `ModelVersion` at all?

Reply:
updated docstrings according to the summary line style guide. 

as for making those attributes private, i do think these values should probably be made private, as it doesn't seem like they would be useful for anything. i was initially hesitant because they are currently accessible as part of our public api, so this would be a breaking change, but i think it's worth doing. so i removed these from the ModelVersion, and i've added a line to our SDK confluence doc to track breaking changes.